keps/13243-multikueue-ray-worker-autoscaling/README.md

```
@@ -0,0 +1,342 @@
+# KEP-13243: MultiKueue Worker-Side Ray In-Tree Autoscaling for Elastic Jobs
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1: Online/offline inference](#story-1-onlineoffline-inference)
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)